Cargo.toml.jinja

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "frequenz-{{project_name}}"
+description = "{{project_description}}"
+version = "0.1.0"
+edition = "2021"
+authors = ["Frequenz Energy-as-a-Service GmbH"]
+{% if open_source %}
+license = "MIT"
+{% else %}
+license = "proprietary"
+{% endif %}
+
+{% if project_type == "lib" %}
+[lib]
+name = "frequenz_{{project_name | replace("-", "_")}}"
+path = "src/lib.rs"
+{% if python %}
+crate-type = ["cdylib", "rlib"]
+{% endif %}
+{% endif %}
+
+{% if project_type == "bin" %}
+[[bin]]
+name = "frequenz_{{project_name | replace("-", "_")}}"
+path = "src/main.rs"
+{% endif %}
```

Comment:
This feels inconsistent. We are adding templates for binaries here, but at the same time ignoring Cargo.lock in gitignore.

Reply:
Are we planning to add binaries to this github organization? If not, is there a reason to add templates for this now?

Reply:
no binaries :)